src/lib/pubcidModule.js

```diff
@@ -2,44 +2,109 @@ import PubcidHandler from './pubcidHandler';
 import {uuid4} from "./utils";
 
 /**
- * Create a window level PublisherCommonId object that sets and returns pubcid (aka fpc).
- * @param {Window} w window object
- * @param {Document} d document object
+ * Create a window level PublisherCommonId object that sets and returns pubcid.
+ * @param {Window} win window object
+ * @param {Document} doc document object
  * @param {object} options Additional options.
  */
-export function setupPubcid(w, d, options = {}) {
+export function setupPubcid(win, doc, options = {}) {
 
-    const PublisherCommonId = (function() {
-        if (typeof w.PublisherCommonId === 'object') {
-            return w.PublisherCommonId;
+    const name = 'PublisherCommonId';
+    const delegate = win[name] || {};
+    delegate.que = delegate.que || [];
+
+    /**
+     * Process one of the queued commands.  This is also mapped to PublisherCommonId.que.push.
+     * @param {Object[] | function} args An array or a function.  If it's an array, then the first item is the method
+     * name, followed by parameters.
+     */
+    function processQueue(args) {
+        if (typeof args !== 'function') {
+            const params = [].slice.call(args);  // convert to a proper array
+            const method = params.shift();       // separate method name from the rest
+            if (typeof delegate[method] === 'function') {
```

Comment:
You could log something if you ever hit the else condition, otherwise it looks like whatever you're processing will be lost.  Not necessarily an error, maybe just info

Reply:
Good idea.  I'll add some logging for the queue.